### ISSUE TITLE
SQL Translation Error

### ISSUE THREAD

**Issue by sparty02** [jOOQ/jOOQ#7295]
### SQL translator incorrectly translates this query:

```sql
CREATE TABLE employees
( employee_id INT NOT NULL,
  last_name VARCHAR(50) NOT NULL,
  first_name VARCHAR(50),
  salary MONEY
)
```

Parser output:

```sql
Unknown data type: [5:10] ...(50) NOT NULL,
  first_name VARCHAR(50),
  salary [*]MONEY
);
```

(referenced from https://www.techonthenet.com/sql_server/tables/create_table.php)

**Comment by lukaseder** [comment]
Thanks for reporting this. We're probably not going to support the `MONEY` type in jOOQ. In both databases I'm aware of that support the type, it is discouraged to use it:

- SQL Server: https://stackoverflow.com/a/582819/521799
- PostgreSQL: https://stackoverflow.com/q/15726535/521799

**Comment by lukaseder** [comment]
This is fixed in jOOQ 3.12 through #7776. Even if such custom data types still cannot be translated between dialects, they can now be parsed.
